### PR TITLE
Update README.md

### PR DESCRIPTION
**Pull request by bagipro**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -10,10 +10,11 @@ A collection of android security related resources.
 
 ### Online Analyzers
 
-1. [AndroTotal](http://andrototal.org/)
-5. [Appknox](https://www.appknox.com/) - not free
-8. [AVC UnDroid](http://undroid.av-comparatives.info/)
-10. [Virustotal](https://www.virustotal.com/) - max 128MB
+1. [Oversecured](https://oversecured.com/) - Enterprise vulnerability scanner for Android and iOS apps, it offers app owners and developers the ability to secure each new version of a mobile app by integrating Oversecured into the development process. Not free.
```
Comment (ashishb) [inline README.md:13]: Why move this to the top?

File: README.md [hunk 2]
```diff
@@ -244,6 +254,14 @@ A collection of android security related resources.
 7. [Vulnerability Google Doc](https://docs.google.com/spreadsheet/pub?key=0Am5hHW4ATym7dGhFU1A4X2lqbUJtRm1QSWNRc3E0UlE&single=true&gid=0&output=html)
 8. [Google Android Security Team’s Classifications for Potentially Harmful Applications (Malware)](https://source.android.com/security/reports/Google_Android_Security_PHA_classifications.pdf)
 
+### Vulnerabilities
```
Comment (ashishb) [inline README.md:257]: Please remove this section. It is a pure advertisment.

File: README.md [hunk 3]
```diff
@@ -229,7 +230,16 @@ A collection of android security related resources.
 3. [android app security checklist](https://github.com/b-mueller/android_app_security_checklist)
 4. [Mobile App Pentest Cheat Sheet](https://github.com/tanprathan/MobileApp-Pentest-Cheatsheet)
 5. [Android Reverse Engineering 101 by Daniele Altomare (Web Archive link)](http://web.archive.org/web/20180721134044/http://www.fasteque.com:80/android-reverse-engineering-101-part-1/)
-5. ~~[Mobile Security Reading Room](https://mobile-security.zeef.com) - A reading room which contains well categorised technical reading material about mobile penetration testing, mobile malware, mobile forensics and all kind of mobile security related topics~~
+6. [Android security checklist: WebView](https://blog.oversecured.com/Android-security-checklist-webview/)
```
Comment (ashishb) [inline README.md:233]: This repo is not meant as an advertisement for oversecured.com

**Review by ashishb — CHANGES_REQUESTED** [review]

File: README.md (hunk 3 above)
Comment (bagipro) [reply README.md:233]: Hey @ashishb,

The previous item is an article but not a collection of articles. I initially wanted to add something like "[Oversecured blogs](https://blog.oversecured.com/)" to this section, but I considered this to be an advertisement.

How would you recommend noting the articles? They consider cool technical write-ups regarding Android security.

Thanks!

**Comment by ashishb** [comment]
> How would you recommend noting the articles? They consider cool technical
write-ups regarding Android security.

The blog list can be endless. So, they don't belong to this repo.

Ashish

File: README.md (hunk 1 above)
Comment (bagipro) [reply README.md:13]: I've moved it from one category to another and updated the description, and placed it at the top as recently updated

**Pull request closed without merging**